README.md

```diff
@@ -124,6 +124,12 @@ profile_slurm::files::files:
   ...
 ```
 
+A mailprog.sh script can be set up which, when configured as MailProg,
```

Comment:
Perhaps we want to mention a bit more about this `sendas_address` email address. e.g. 
- Does it need to be a valid email address?
- Any tips on setting up a valid email address for this usage? e.g. URL to those tips.

Reply:
Thanks! I've added to README.md to address those questions/suggeset